students/jeonghyun-ahn/westagram/users/views.py

```diff
@@ -0,0 +1,37 @@
+import json, re
+from .models            import User
+from django.db.models   import aggregates
+from django.http        import JsonResponse
+from django.views       import View
+
+class UserView(View):
+    def post(self, request):
+        data        = json.loads(request.body)
+        id_1        = data['email']
+        input2      = data['password']
```

Comment:
id_1, input2는 어떤 변수를 나타내는지 불명확합니다.
```suggestion
        email        = data['email']
        password     = data['password']
```